kolibri/plugins/user_auth/assets/src/modules/signUp/handlers.js

```diff
@@ -1,14 +1,17 @@
 import FacilityResource from 'kolibri-common/apiResources/FacilityResource';
+import useFacilities from 'kolibri-common/composables/useFacilities';
 import { ComponentMap } from '../../constants';
 
 export function showSignUpPage(store, fromRoute) {
+  const { setFacilities } = useFacilities();
+
   // Don't do anything if going between Sign Up steps
   if (fromRoute.name === ComponentMap.SIGN_UP) {
     return Promise.resolve();
   }
   return FacilityResource.fetchCollection()
```

Comment:
Hrm, do you know if there's a reason we don't just use `getFacilities` here?

Reply:
The existing functionality was using set facilities, I just replaced it. Are you suggesting that instead of calling the fetchCollection we directly call getFacilities here?

Reply:
ohh yes! it does the same thing will replace!